### PR TITLE
Added Tutorial: Gloo Edge for #3199

### PR DESCRIPTION
**Pull request by Parsifal-M**
Signed-off-by: Peter Macdonald <macdonald.peter90@gmail.com>

Hey @ashutosh-narkar I've added the tutorial as requested, I did some very small changes to some of the wording (i hope you don't mind). Also I noticed it doesn't have a "Wrapping Up" section, is this something you'd like to add? if you have something in  mind I can add this also.

Please let me know your feedback? :smile: 

Thanks!

### CODE REVIEW

File: docs/content/envoy-tutorial-gloo-edge.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+}
+
+action_allowed {
+  http_request.method == "GET"
+}
+```
+
+Store the policy in Kubernetes as a Secret.
+
+```bash
```
Comment (ashutosh-narkar) [inline docs/content/envoy-tutorial-gloo-edge.md:124]: We should host the policy on a local bundle server. See [this](https://www.openpolicyagent.org/docs/latest/envoy-tutorial-standalone-envoy/#4-publish-opa-bundle) for reference.

File: docs/content/envoy-tutorial-gloo-edge.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+              name: opa-policy
+          args:
+            - "run"
+            - "--server"
+            - "--addr=0.0.0.0:8181"
+            - "--set=plugins.envoy_ext_authz_grpc.addr=0.0.0.0:9191"
+            - "--set=plugins.envoy_ext_authz_grpc.query=data.envoy.authz.allow"
+            - "--set=decision_logs.console=true"
+            - "--ignore=.*"
+            - "/policy/policy.rego"
```
Comment (ashutosh-narkar) [inline docs/content/envoy-tutorial-gloo-edge.md:164]: Update this to pull policy from the bundle server. See [this](https://www.openpolicyagent.org/docs/latest/envoy-tutorial-standalone-envoy/#5-create-app-deployment-with-opa-and-envoy-sidecars) for reference.

**Comment by ashutosh-narkar** [comment]
@Parsifal-M thanks for working on this. It would be great if we had a `Wrap Up` section like the other tutorials.

**Comment by Parsifal-M** [comment]
> @Parsifal-M thanks for working on this. It would be great if we had a `Wrap Up` section like the other tutorials.

Hey @ashutosh-narkar 

Will check on your reviews and add a Wrap Up section :+1: 

Thanks!

**Comment by Parsifal-M** [comment]
Hey @ashutosh-narkar just FYI I have not forgotten, I'm on holiday this week but I did bring my laptop to do some cool bits and pieces, so I will post back later tonight I think 👍

**Comment by Parsifal-M** [comment]
Hello @ashutosh-narkar

Sorry for the delay on this :smile: I actually noticed I missed a small section while testing this as well which I have corrected.

Tested the whole tutorial locally and I seem to get the correct results.

Please let me know if we need to update anything further? I have squashed the commits also.

Thanks!

**Comment by Parsifal-M** [comment]
Need to sort out my branch sorry, will commit again in a bit...

**Pull request closed without merging**